modules/oadp-release-notes-1-2-2.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-2-2_{context}"]
+= OADP 1.2.2 release notes
+
+
+[id="new-features-1-2-2_{context}"]
+== New features
+
+There are no new features in the release of {oadp-first} 1.2.2.
+
+// :FeatureName: OADP Data Mover
+// include::snippets/technology-preview.adoc[]
+
+[id="resolved-issues-1-2-2_{context}"]
+== Resolved issues
+
+The following highlighted issues have been resolved in OADP 1.2.2:
+
+
+.Restic restore partially failed due to Pod Security standard
+
+In previous releases of OADP 1.2, {OCP} 4.14 enforced a pod security admission (PSA) policy that hindered the readiness of pods during a Restic restore process.
+
+This issue has been resolved in the OADP 1.1.6 and OADP 1.2.2 releases. Therefore, it is recommended that users upgrade to these releases.
```

Comment:
Is it upgrade to either one of these releases? Or do they need to upgrade to 1.1.6 and then 1.2.2?

Reply:
It has been resolved in 1.1.6 and also 1.2.2